docs/DeveloperGuide.md

```diff
@@ -270,51 +270,189 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* *`    | eager to learn student                 | receive feedback (read comments and suggestions) from my professor                                                                      | I can set a direction to make progress for my FYP [coming soon]               |
+| `* *`    | responsible student                    | update my FYP progress through the app                                                                                                  | professor can be informed on my progress [coming soon]                        |
+| `* *`    | critical student                       | provide feedbacks on my FYP guidance and evaluation                                                                                     | so that I can help the FYP management team to further improve [coming soon]   |
+| `* *`    | helpful SoC professor                  | put tags on my student's FYP project so that I can let him/her know whether they are “Ahead of time”, “Up to task” or “Behind schedule” | they can be informed of their progress relative to the timeline [coming soon] |
+| `* *`    | non-confrontational (i.e. shy) student | push certain tags (e.g. “Assistance required!”, “Unable to complete :(”) to the application                                             | gather my professor’s attention without direct confrontation [coming soon]    |
+| `*`      | fun-seeking professor/FYP student      | end “stickers” (similar to Telegram stickers) or emoji so that the app                                                                  | is more appealing to use [coming soon]                                        |
+| `*`      | FYP student                            | update my availability slots on the app                                                                                                 | my professor knows when I will be available to meet [coming soon]             |
+| `*`      | well-versed SoC professor              | sort my student’s FYP based on what specialisation they are taking                                                                      | I can better organise the FYPs [coming soon]                                  |
+| `*`      | very meticulous SoC professor          | see student's FYP sorted by research area first, followed by alphabetical order                                                         | I can gain quick easy access to the details required [coming soon]            |
 
 *{More to be added}*
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `JerylFypManager` and the **Actor** is the `Professor`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 - Adding FYP**
```

Comment:
How about adding bullet points so that we can see a clear indentation among different use cases? Bolding only might not seem contrast enough to do the job.

Reply:
Should we make it like 

 UC01. **Use case: - Adding FYP**

Reply:
Yeah